### DISCUSSION TITLE
Ostrich Egg Creating a Markdown file from terminal

### DISCUSSION THREAD

**Discussion by skreling** [sr320/course-fish546-2021#39] [Q&A] [answered]
This might be a dumb question, but I can't find any help for it online and didn't find anything super useful in the text book, but I can't figure out how to make a markdown file from terminal within the subdirectory I created in the Ostrich/Documents. Alternatively, I made a markdown file and filled with w text, links, images etc. but then couldn't figure out how to push it from my local machine up to the ostrich's directories?

**Comment by sr320** [comment, marked as answer]
Good Question. A few options, but what I wanted folks to use was `nano` as you did in Week 1. 

See https://github.com/sr320/course-fish546-2021/blob/master/01-Notes/Tutorials/Navigating-commandline.md#creating-things

**Reply by skreling** [reply]
Thanks for the quick reply! So I guess I'm just a little confused. I did originally go to use Nano, but I thought it was just a text editor and even in the file you just linked it says, "When we say, "nano is a text editor," we really do mean "text": it can only work with plain character data, not tables, images, or any other human-friendly media"

So I could create an .rmd file but how would I know if it was functioning? I can call head <file> and take a look at it just appears as text as expected. Secondly, my understanding could be wrong, but you asked us to include an image in the mark down file and I thought for Rmds to display an image the image needs to live in a folder with the .Rmd file. For reference this is what I was going for, but you can see the direct file path in the rmd file that I don't know how to replicate
<img width="877" alt="Screen Shot 2021-01-26 at 12 44 10 PM" src="https://user-images.githubusercontent.com/76884520/105903031-41686c00-5fd4-11eb-80e1-2f42c0ff8334.png">

**Reply by sr320** [reply]
Correct you cannot view. But you can create a `.md` file as this is simply a text file. 

and an image in `.md` is done as follows.

`![image-descript](URL)`

or 

`![image-descript](local-file-location)`

Note `.Rmd` is different than simple `.md`.

**Reply by skreling** [reply]
thank you!
